src/config.lisp

```diff
@@ -2,7 +2,7 @@
 
 (defun lem-home ()
   (or (uiop:getenv "LEM_HOME")
-      (merge-pathnames ".lem/" (user-homedir-pathname))))
+      (uiop:xdg-config-home "lem/")))
```

Comment:
This should fall back to using `~/.lem` so that current users aren't caught off-guard, right?

Reply:
I changed the logic to take .lem if it exists first and fallback to xdg-config + 'lem/' if that doesn't already exist. Thoughts?